.github/workflows/ci.yml

```diff
@@ -148,6 +150,7 @@ jobs:
     name: Model Tests
     runs-on: ubuntu-latest
     needs: build
+    timeout-minutes: 90
```

Comment:
Am I correct in thinking that model tests profiles depend on what was changed? Or, are all profiles always executed?

Reply:
All profiles are always executed. There is no straightforward way to map profiles to changes yet.